test/test_sparse.py

```diff
@@ -738,7 +738,7 @@ def test_cross_device(x1, x2):
         self.assertEqual(None, x1.grad)
 
     @onlyCUDA
-    def test_cuda_empty(self, _):
+    def test_cuda_empty(self):
```

Comment:
This should still take and use the device string because the device string represents an actual device, and not a device type. It could be cuda:0 or cuda:1, for example. They should not be calling `x.cuda(0)`.

Reply:
Fixed here and in the test below